app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/requests/EISAmendCaseRequest.scala

```diff
@@ -37,13 +36,13 @@ case class EISAmendCaseRequest(
 object EISAmendCaseRequest {
   implicit val formats: Format[EISAmendCaseRequest] = Json.format[EISAmendCaseRequest]
 
-  case class Content(CaseID: String, Description: String, EORI: Option[EORI])
+  case class Content(CaseID: String, Description: String)
 
   object Content {
     implicit val formats: Format[Content] = Json.format[Content]
 
-    def from(request: AmendClaimRequest, eori: Option[EORI]): Content =
-      Content(CaseID = request.Content.CaseID, Description = request.Content.Description, EORI = eori)
+    def from(request: AmendClaimRequest): Content =
+      Content(CaseID = request.Content.CaseID, Description = request.Content.Description)
```

Comment:
OBSERVATION: We know/knew the EORI number here.  Would it not be easier to prefix the Description with the EORINumber here, and avoid any FE changes?
`Content(CaseID = request.Content.CaseID, Description =s"[EORINumber=$eori] ${request.Content.Description}")`

Reply:
Being consistent with Create Claim mechanism.  I will approved then.